engine/src/main/java/org/hibernate/validator/internal/engine/messageinterpolation/el/SimpleELContext.java

```diff
@@ -25,7 +24,6 @@
 	private static final ELResolver DEFAULT_RESOLVER = new CompositeELResolver() {
 		{
 			add( new RootResolver() );
-			add( new StaticFieldELResolver() );
```

Comment:
I understand this was the purpose of this PR, but it seems odd that a feature is removed, and no documentation is removed. Was this undocumented?

Reply:
Yes, it's related to the javax.el upgrade: it's a new feature in javax.el so it was exposed as we expose all the others. We don't detail what's possible with javax.el though.